clouddriver-docker/src/main/groovy/com/netflix/spinnaker/clouddriver/docker/registry/api/v2/client/DockerRegistryClient.groovy

```diff
@@ -458,6 +463,8 @@ class DockerRegistryClient {
         } else {
           throw error
         }
+      } catch (SpinnakerServerException error) {
+        throw error
```

Comment:
is there some groovy reason why this is needed? Catching and immediately rethrowing serves no purpose that I can see?

(See also `DockerRegistryNamedAccountCredentials` and `LocalFileUserDataProvider`)